src/layouts/default.vue

```diff
@@ -61,6 +62,16 @@ onBeforeUnmount(() => {
 });
 
 useHead({
+  link: [
+    { rel: 'icon', href: '/favicon.svg', type: 'image/svg+xml', },
+    { rel: 'apple-touch-icon', href: '/apple-touch-icon.png', },
+  ],
+  meta: [
+    {
+      name: "google-site-verification",
+      content: "zpj1hb-uMNGkFLJlxqC96oWE067G1lzXcRFsWM4m_8M",
+    },
+  ],
```

Comment:
I wonder if there's a reason to put it here and not in nuxt config (have the same question about the script below as well in fact :) )